js/app.js

```diff
@@ -1,23 +1,113 @@
-"use-strict"; 
+"use-strict";
 
-// let totalAnswers=25
-// let currentAnswers=0
+// console.log("hello world");
+//global variables
+let totalAnswers = 25;
+let currentAnswers = 0;
 
-let imgBody = document.getElementById ("img-Body");
-let imgOne = document.getElementById ("img-one");
-let imgTwo = document.getElementById ("img-two");
-let imgThree = document.getElementById ("img-three"); 
+function Product(name) {
+  this.name = name;
+  this.votes = 0;
+  this.views = 0;
+}
+
+Product.prototype.render = function (i) {
+  let img = document.getElementById(`img-${i}`);
+  img.src = `img/${this.name}.jpg`;
```

Comment:
Appending the image path later is an interesting choice. Great for this project, but less good with larger scale.